less/index.less

```diff
@@ -105,7 +131,7 @@ html, body {
 	}
 
 	p{	justify-content: center;
-		color:#C2DFFF;
+		.header-font;
```

Comment:
Good job using a mixin! You can also refactor this like instead of keep on repeating the mixin, you can:
```
h1, h2, p {
  .header-font;
}
```
to apply `.header-font` to h1, h2 and p.